### PR TITLE
Load tests Jenkins job

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
Add Jenkins job for load tests. Also remove using ffmpeg from load tests to decrease memory usage.

### CODE REVIEW

**Review by dmytro-ndp — CHANGES_REQUESTED** [review]
[There is a problem with screencast recording](https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/che-pr-tests/view/K8S/job/Multiuser-Che-PR-check-E2E-Happy-path-tests-against-k8s/2833/console):
```
09:14:36.745 INFO [ActiveSessions$1.onStop] - Removing session b0955a780b82852a8908b46e73f6033e (org.openqa.selenium.chrome.ChromeDriverService)
+ EXIT_CODE=0
+ kill_ffmpeg
+ echo 'Killing ffmpeg with PID='
+ kill -2 ''
Killing ffmpeg with PID=
/tmp/entrypoint.sh: line 7: kill: `': not a pid or valid job spec
+ wait ''
/tmp/entrypoint.sh: line 8: wait: `': not a pid or valid job spec
+ mkdir -p /tmp/e2e/report/
+ cp '/tmp/ffmpeg_report/*' /tmp/e2e/report/
cp: cannot stat '/tmp/ffmpeg_report/*': No such file or directory
+ exit 0
```
Please, fix it before merging the changes into the **master** branch.

File: tests/.infra/crw-ci/load-tests/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                sh """
+                    set +x
+                    echo "Installing oc..."
+                    wget -q https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.3.1/openshift-client-linux-4.3.1.tar.gz -O - | tar -xz
+                    mv oc $OC_PATH
+                    echo "Setting CHE context..."
+                    oc login $cheClusterUrl -u $cheClusterUsername -p $cheClusterPassword --insecure-skip-tls-verify
+                    oc project $cheClusterNamespace
+                """
+                script {
```
Comment (dmytro-ndp) [inline tests/.infra/crw-ci/load-tests/Jenkinsfile:83]: you can enclose whole _steps_ section into _script_{} block,
or have single sh block with bash commands like `CHE_CONTEXT=${oc whoami -c}` inside

File: tests/.infra/crw-ci/load-tests/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+    options {
+        timestamps()
+        timeout(time: 3, unit: 'HOURS')
+        buildDiscarder(logRotator(artifactDaysToKeepStr: '',
+                artifactNumToKeepStr: '', daysToKeepStr: '60', numToKeepStr: '30'))
+    }
+
+    parameters {
+        string(name: 'emails',
```
Comment (dmytro-ndp) [inline tests/.infra/crw-ci/load-tests/Jenkinsfile]: `emails` parameter is unused in pipeline

File: tests/.infra/crw-ci/load-tests/Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                sh"""
+                  echo "Clone che"
+                  git clone $repoWithScripts
+                """
+            }
+        }
+        stage('Run load tests') {
+            steps {
+                sh """
+                  ROOT=`pwd`
```
Comment (dmytro-ndp) [inline tests/.infra/crw-ci/load-tests/Jenkinsfile:122]: Actually, it's not simple to read to code ten rows long.
I would suggest to divide logical blocks by empty rows to improve readability.

**Comment by dmytro-ndp** [comment]
crw-ci-test

**Comment by Katka92** [comment]
@dmytro-ndp The problem is fixed: https://github.com/eclipse/che/pull/16992#issuecomment-632662318 I've reverted the image.

**Review by dmytro-ndp — APPROVED** [review]
Screencast problem has been fixed https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Multiuser-Che-PR-check-E2E-Happy-path-tests-against-k8s/2836/console

**Review by dmytro-ndp — APPROVED** [review]
Latest changes look good.

**Comment by Katka92** [comment]
[ci-build]

**Pull request merged**